pydotorg/urls.py

```diff
@@ -1,71 +1,72 @@
-from django.conf.urls import include, url, handler404
```

Comment:
I'm not talking about line 50 or structure of the change. Most of the changes in this file are only whitespace changes which I think caused by different editor configuration. See the diff between https://github.com/python/pythondotorg/pull/894/files (with whitespaces) and https://github.com/python/pythondotorg/pull/894/files?w=1 (without whitespaces)